package.json

```diff
@@ -47,7 +47,10 @@
     "split": "yarn process web split",
     "svgpdf": "./scripts/svg_to_pdf.sh",
     "test": "node ./scripts/test.js",
-    "try": "cd docs_out; http-server --cors --port 8080"
+    "try": "cd docs_out; http-server --cors --port 8080",
+    "via": "yarn do via",
+    "web": "yarn process web",
```

Comment:
Are these changes related to your PR?

Reply:
These lines are likely to come from a manual merge; some version issue on Hung's side: I have removed these lines some time ago and don't want them back :-)

Reply:
Yes prof, it is probably my fault :-(